src/components/tutorials/getting-started-with-oauth2.js

```diff
@@ -17,10 +17,14 @@ class GettingStartedUsingQppOauth2 extends React.Component {
         <p className='ds-text'>Once you've successfully completed your integration in the Developer Preview, you can request access to the production Submissions API. We will provide details on this production access process this fall.</p>
 
         <h1 className='ds-h1' id='oauth'>Using OAuth in the Developer Preview</h1>
-        <p className='ds-text'>To use the QPP Submissions API with OAuth, a developer must first register their application in the Developer Preview.</p>
-        <p className='ds-text'>A registered application is given a client ID and a client secret for this testing environment. The secret should only be used if it can be kept confidential, such as communication between your server and the Submissions API.</p>
-        <p className='ds-text'>Registering and testing your application within the Developer Preview is required prior to being granted production Submissions API access.</p>
-        <p className='ds-text'>You can register your application at <a href='https://preview.qpp.cms.gov/'>https://preview.qpp.cms.gov/</a>.</p>
+        <p className='ds-text'>To use the QPP Submissions API with OAuth, a developer must first create an account in Developer Preview, and then register their application.</p>
+        <p className='ds-text'>Developer Preview accounts are separate from the ones used to log in to the main QPP website. To create a Developer Preview account, please follow the process at <ahref='https://impl.harp.qualitynet.org/register/profile-info'>https://impl.harp.qualitynet.org/register/profile-info/</a>.</p>
+        <p className='ds-text'>Once you’ve created this account (identify verification is required), you will be able to log in to our Developer Preview environment at <ahref='https://preview.qpp.cms.gov'>https://preview.qpp.cms.gov/</a>.</p>
```

Comment:
Should be a space between a & href for this and the two below.

Reply:
ooh thanks! I think I fixed it in the next committed change? if that was not the way to do this let me know!